gatsby/config/gatsby-browser.tsx

```diff
@@ -7,19 +7,24 @@ export const wrapPageElement = ({ element }) => {
   return <Layout>{element}</Layout>;
 };
 
-const appInsights = new ApplicationInsights({
-  config: {
-    instrumentationKey: process.env.GATSBY_APPINSIGHTS_KEY,
-  },
-});
+const appInsights =
```

Comment:
1. proc jenom production?
2. potrebuje zmenit taky https://github.com/cesko-digital/covid.gov.cz/blob/master/gatsby/src/hooks/useAnalytics.ts

Reply:
1. Není důvod aby byly insights na developmentu, akorát by to zbytečně dělalo zmatek v datech, případně by si každý musel vytvořit svůj klíč aby mohl sbírat jen svoje data, což mi přijde nepraktické.
2. Výsledek bude prostě null, není potřeba měnit, akorát při použití se bude muset udělat check jestli je null.

Reply:
1. 🤷‍♀️
2. ok, tak potrebuje update https://github.com/cesko-digital/covid.gov.cz/blob/master/gatsby/src/types/global.d.ts

Reply:
treba

```
// AnalyticsAdapter.ts
class AnalyticsAdapter {
  private static instance: AnalyticsAdapter;
  private ai: AppInsight | null

  private constructor() {
    this.ai = window.ai
  }

  public static getInstance(): AnalyticsAdapter {
    if (!AnalyticsAdapter.instance) {
      AnalyticsAdapter.instance = new AnalyticsAdapter();
    }

    return AnalyticsAdapter.instance;
  }

  event(opts: AppInsightOptions) {
    this.ai?.trackEvent(opts)
  }
}

// useAnalytics.ts

function useAnalytics() {
  const ai = useState(AnalyticsAdapter.getInstance())
  return ai
}
```